src/index.d.ts

```diff
@@ -544,6 +544,48 @@ declare namespace admin.auth {
     toJSON(): Object;
   }
 
+  /**
+   * Interface representing the common properties of a user enrolled second factor.
+   */
+  interface MultiFactorInfo {
+  
+    /**
+     * The unique identifier of the enrolled second factor.
```

Comment:
Nit: This is the user's unique ID from the 2nd factor right? Perhaps make that a bit clear.

Reply:
Done.